packages/common/src/lib/common/tools.ts

```diff
@@ -2047,15 +2098,15 @@ function getConfigFileName() {
 
     // if debugging with npm5 -> node_modules on e.g. /opt/node_modules
     if (
-        fs.existsSync(`${__dirname}/../../../../../../../node_modules/${appName.toLowerCase()}.js-controller`) ||
-        fs.existsSync(`${__dirname}/../../../../../../../node_modules/${appName}.js-controller`)
+        fs.existsSync(`${__dirname}/../../../../../../../../node_modules/${appName.toLowerCase()}.js-controller`) ||
+        fs.existsSync(`${__dirname}/../../../../../../../../node_modules/${appName}.js-controller`)
```

Comment:
🤮

Reply:
if debugging with npm 5.. not sure if this is still needed?